node/src/Utils.h

```diff
@@ -233,6 +244,29 @@ namespace node {
         return true;
     }
 
+    inline bool GetFloatArray(const Napi::Value& jsValue,
+                              std::vector<float>& array,
+                              const size_t size = std::numeric_limits<size_t>::max()) {
```

Comment:
There are duplicated codes with `GetInt32Array`. Could we make a template function with data type?

Reply:
I have updated as you suggested, PTAL, thanks! @huningxin